app/src/main/java/ch/epfl/sdp/blindly/Profile6.kt

```diff
@@ -1,11 +1,18 @@
 package ch.epfl.sdp.blindly
 
+import android.content.Intent
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
+import android.view.View
 
 class Profile6 : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.set_profile_6)
     }
+    fun start_profile_7(view: View) {
+        // Do something in response to button
```

Comment:
I think this comment could be deleted.

Reply:
Yes you are right, I will delete this in the next PR.